app/src/test/java/ar/scacchipa/twittercloneapp/viewmodel/AuthWebDialogViewModelTest.kt

```diff
@@ -81,6 +71,15 @@ class AuthWebDialogViewModelTest {
         Assert.assertTrue(
             subject.tokenResource.value == TokenResource.Cancel)
     }
+    @Test
+    fun webViewShouldWebResourceErrorToSubject() {
```

Comment:
something is missing in the name of the test to be clear, maybe a verb after should

Reply:
Now, the function is called webViewSendWebResourceErrorToSubject()